cmd/gpu_plugin/README.md

```diff
@@ -36,11 +37,23 @@ For example containers with Intel media driver (and components using that), can
 video transcoding operations, and containers with the Intel OpenCL / oneAPI Level Zero
 backend libraries can offload compute operations to GPU.
 
+Intel GPU plugin may register four node resources to the Kubernetes cluster:
```

Comment:
Our top-level README has the "workload authors" section for this type of info. Can you add the equivalent content there?

Reply:
Yep, I'll do that.